src/container/srv_target.c

```diff
@@ -2537,3 +2660,114 @@ ds_cont_ec_timestamp_update(struct ds_cont_child *cont)
 {
 	cont->sc_ec_update_timestamp = d_hlc_get();
 }
+
+void
+ds_cont_child_insert_dtx(struct ds_cont_child *cont_child, struct dtx_leader_handle *new_dlh)
```

Comment:
According to our discussion: when DTX resync happened, the VOS level DTX iteration will return all the active (prepared but not committed) DTX entries, during such iteration, we can very easily find out the one with oldest epoch_X. All updates with older epoch must have been either committed or aborted, do not need to care, so the incremental rebuild can be started from such epoch_X. With such method, we do not need to maintain the relative expensive active DTX list during every update. That will much simplify the  incremental reintegration.